page-objects/createCourse.js

```diff
@@ -80,5 +126,191 @@ module.exports = {
 		const helper = chosenSearchableSelectHelper(driver, courseData.elem.selectorWithMultipleChoiceStudents);
 		await helper.selectOptionByName(name);
 		await this.goToNextSectionCreateCourse();
+	},
+	areSelectorsOnThePage: async function() {
+	let isImportCourseBtnOnThePage = await helpers.isSelectorOnThePage(courseData.elem.importCourseBtn);
+	await expect(isImportCourseBtnOnThePage).to.equal(1);
+	let isCreatedCourseBtnOnThePage = await helpers.isSelectorOnThePage(courseData.elem.createCourseBtn);
+	await expect(isCreatedCourseBtnOnThePage).to.equal(1);
+
+	},
+	clickCreateCourseBtn: async function()  {
+		let btn = await driver.$(courseData.elem.createCourseBtn);
+		await btn.click();
+		await driver.pause(1000);
+	},
+	canProceedToStage: async function(stageNum) {
+		switch (stageNum) {
+			case 2: let selector = await driver.$(courseData.elem.stage1Selector);
+			let children = await selector.$$(".//*");
+			if (children.length>0) {
```

Comment:
There's no need for an if-statement here.
You could:
```
return children.length>0
```
Which will be either `true` or `false`